src/current/cockroachcloud/free-trial.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Try CockroachDB Cloud for free [WIP]
+summary: Learn about free trial credits for CockroachDB Cloud
+toc: true
+cloud: true
+--- 
+
+When you sign up for CockroachDB Cloud, your new organization's account starts with $400 in free credits, allowing you to try the product at no cost.
+
+## Free trial credits
+
+Free trial credit is applied to the first $400 in costs for all clusters created in your organization, regardless of the deployment type: CockroachDB Standard, Advanced, or Basic.
+
+Monthly billing begins when the free trial ends: on the day that the last credit is used, or upon expiration of credit, if any remains. By the end of the trial, you must provide a credit card for continued billing to prevent interruptions in service or loss of data.
+
+### Eligibility
+
+For a user who signs up with CockroachDB Cloud, only their first organization is eligible for free trial credits. Users invited to an organization do not qualify for additional credits. If provided with necessary permissions, they can create clusters that will share the organization's existing credits.
```

Comment:
I'm confused about "user" here. I think it is per organization. I don't think users who are invited get more credit than the initial $400. Maybe worth confirming.

Reply:
This is trying to say that if you invite a user to your org after you initiate a trial, your org still has the same number of trial credits. It's per org, not per user. Could probably be clearer. Would like to revisit after this PR.

Reply:
Got it, gave a suggested edit below but I won't apply it for now.